music/src/core/sequences.ts

```diff
@@ -402,6 +402,19 @@ export function unquantizeSequence(qns: INoteSequence, qpm?: number) {
   return ns;
 }
 
+/**
+ * Create an empty quantized NoteSequence with steps per quarter note and tempo.
+ * @param stepsPerQuarter The number of steps per quarter note to use.
+ * @param qpm The tempo to use.
+ * @returns A new quantized NoteSequence.
+ */
+export function createQuantizedNoteSequence(
```

Comment:
this doesn't feel like a helper here (it's too generic), maybe just in the class you're using it in?

Reply:
But now it's being used in two places: melodies.ts and data.ts.

Reply:
👍